### DISCUSSION TITLE
Service is running but not listening Error: 502

### DISCUSSION THREAD

**Discussion by Emekaborisama** [okteto/okteto#1561] [General]
I want to deploy a gitlab project via Okteto cloud and i am using the git deploy feature with the manifest script below. 

- okteto-stack.yaml

`name: firstpro
services:
  firstpro:
    public: true
    image: okteto.dev/firstpro
    build: .
    replicas: 1
    ports:
      - 8080
    resources:
      cpu: 2000m
      memory: 2280Mi`

i am getting `Service is running but not listening Error: 502` error on the interface  and `container 'firstpro' failed with exit code 137.` error on the log

**Comment by pchico83** [comment]
@Emekaborisama looks like your service process is not starting properly, you need to add logs to troubleshoot why it fails to start.
Also, for Okteto Cloud related issues, it is better to use this repo:
https://github.com/okteto/cloud-feedback
